origin-bridge/src/env.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
```

Comment:
Just curious, is there a way to set environmental variables without having `envKey` running locally?

Reply:
You can just export each directly, e.g. `export TWILIO_VERIFY_API_KEY=abcd123` or you can `cp dev.env .env` and fill them out in there which should be read by dotenv when the package starts.

Reply:
ok tnx 👍